### PR TITLE
Fix expansions location

### PR DESCRIPTION
**Pull request by marekjeszka**
I think expansions should be in _Info_ tab instead of  _Descr_ tab.

Open points (unclear or with need for help):
- [ ] is it okay to add `com.boardgamegeek.ui.widget.GameDetailRow` into `fragment_game.xml` or it should be extracted to separate file like other included layouts above?
- [ ] `onListQueryComplete` method is now duplicated, should I extract it to some common class?
- [ ] colour of the icon for extensions doesn't match other icons on the _Info_ tab (it's gray instead of blue-ish), is there a setting to make the colour align?

### CODE REVIEW

**Comment by ccomeaux** [comment]
> I think expansions should be in _Info_ tab instead of _Descr_ tab.

They should be on the _Credits_ tab. Whether they stay there or move to a different tab is subjective. BGG has a separate _Expansions_ tab, but I've received feedback there are too many tabs already. I'll see how it looks before merging the PR. 

> Open points (unclear or with need for help):
> 
> * [ ]  is it okay to add `com.boardgamegeek.ui.widget.GameDetailRow` into `fragment_game.xml` or it should be extracted to separate file like other included layouts above?

They way you've implemented it is fine.

> * [ ]  `onListQueryComplete` method is now duplicated, should I extract it to some common class?

No, I don't think there's enough code there to worry about it.

> * [ ]  colour of the icon for extensions doesn't match other icons on the _Info_ tab (it's gray instead of blue-ish), is there a setting to make the colour align?

The `colorize` method handles this. You already modified the credits tab to remove it. You just need to add it to the `GameFragment`.

**Comment by marekjeszka** [comment]
> They should be on the _Credits_ tab. Whether they stay there or move to a different tab is subjective. BGG has a separate _Expansions_ tab, but I've received feedback there are too many tabs already. I'll see how it looks before merging the PR.

I agree that new tab shouldn't be added.
I don't think _Credits_ is the correct one, because it suggests that one will find there information about who contributed to the game. Expansions are more like general information about a game, similar to complexity or amount of players.

> The `colorize` method handles this. You already modified the credits tab to remove it. You just need to add it to the `GameFragment`.

Thanks for the hint, I've added the code to handle that.

File: app/src/main/java/com/boardgamegeek/ui/GameCreditsFragment.kt [hunk 1]
```diff
@@ -84,7 +80,7 @@ class GameCreditsFragment : Fragment() {
     private fun colorize(@ColorInt iconColor: Int) {
         if (!isAdded) return
 
-        listOf(game_info_designers, game_info_artists, game_info_publishers, game_info_categories, game_info_mechanics, game_info_expansions, game_info_base_games)
+        listOf(game_info_designers, game_info_artists, game_info_publishers, game_info_categories, game_info_mechanics)
```
Comment (ccomeaux) [inline app/src/main/java/com/boardgamegeek/ui/GameCreditsFragment.kt]: Move this same call for the new rows into the `GameFragment`. The code should already exist - just need to add the new view groups.

Comment (marekjeszka) [reply app/src/main/java/com/boardgamegeek/ui/GameCreditsFragment.kt]: I don't understand what do you mean. I guess you are referring to this [part](https://github.com/ccomeaux/boardgamegeek4android/pull/124/files#diff-53926588443e39a8c6f3d339a54abfa3R125-R126). Is something missing there?

**Pull request merged**